src/Illuminate/Queue/Worker.php

```diff
@@ -148,7 +148,7 @@ protected function daemonShouldRun()
      */
     protected function waitForChildProcess($processId, $timeout)
     {
-        declare(ticks=1) {
+        declare (ticks = 1) {
```

Comment:
Oh my bad, I read it a while back and didn't notice it was removed. 
Alright won't run php-cs-fixer.